base/models/models.go

```diff
@@ -12,93 +12,80 @@ func (RhAccount) TableName() string {
 }
 
 type SystemPlatform struct {
-	ID               int
-	InventoryID      string
-	RhAccountID      int
-	FirstReported    time.Time
-	S3Url            string
-	VmaasJson        string
-	JsonChecksum     string
-	LastUpdated      time.Time
-	UnchangedSince   time.Time
-	LastEvaluation   *time.Time
-	OptOut           bool
-	ErrataCountCache int
```

Comment:
it looks like ErrataCountCache was removed as well. Just wanted to confirm this was intentional and not missed as part of the rename.

Reply:
Renamed to AdvisoryCountCache

Reply:
ah i see it now, thx